worlds/shivers/requirements.txt

```diff
@@ -0,0 +1 @@
+python-dateutil>=2.9.0.post0
```

Comment:
adding this requirement for calculating the years since a date seems... excessive?

Reply:
removed